docs/topic-guides/blendable-web-apps/html-views.md

```diff
@@ -26,91 +26,97 @@ A partial HTML view is a view that has been nested as a part of a bigger view.
 
 A partial view-model is a view-model that has been nested as a part of a bigger view-model.
 
-A partial view-model together with its corresponding partial HTML view can simply be referred to as a partial.
+A partial view-model together with its corresponding partial HTML view can simply be referred to as a "partial".
 
-## Using partials
+## Rendering HTML views
 
-A partial can be accessed in two different ways:
+A view is rendered when the client obtains a view-model with a view. This can happen in two ways:
 
-1. Directly through an HTTP request that is handled in the application by a `Handle.GET`. The response to this request should be a complete HTML and JSON document. 
-2. Through a blending point. A blending point is a combination of a `Self.GET` call, which acts as the partial view-model blending point, and a `<starcounter-include>` tag, which acts as a partial HTML view blending point. The `<starcounter-include>` tag determines where in the DOM tree the partial HTML view should be rendered.
+1. On a root level, through an HTTP request that is handled in the application by a `Handle.GET`. The response to this request is a view-model. The Starcounter [app shell](app-shell.md) renders it automatically. 
+2. On a nested level, through a pair of insertion points in a view-model and a view:
+   - a blended view-model insertion point is represented by a property in the view-model. The property inserts the result of a `Self.GET` call to a partial view-model in a parent view-model.
+   - a blended view insertion point is represented by a `<starcounter-include>` tag in HTML. The tag inserts the partial view in the parent view.
 
 ## Examples
 
 ### Creating the handler
 
-No matter if the partial is accessed through an HTTP request from the browser or a `Self.GET` call, there always needs to be a handler that deals with the request. This handler returns a partial with the JSON and HTML. The handler, in its simplest form, looks something like this:
+No matter if the view-model is accessed through an HTTP request from the browser or a `Self.GET` call, there always needs to be a handler that deals with the request. This handler returns a view-model. The handler, in its simplest form, looks something like this:
 
 ```csharp
+Application.Current.Use(new HtmlFromJsonProvider());
+Application.Current.Use(new PartialToStandaloneHtmlProvider());
+
 Handle.GET("/your/partial/url", () => 
 {
     return new YourPartialPage();
 }
 ```
 
-This handler only returns the JSON and HTML if the app uses the `HtmlFromJsonProvider` and `PartialToStandaloneHtmlProvider` [middleware](../network/middleware.md).
+The view-model will be rendered by the Starcounter app shell only if the app uses the `HtmlFromJsonProvider` and `PartialToStandaloneHtmlProvider` [middleware](../network/middleware.md).
```

Comment:
Passive form.

The view-model will be rendered by -> The Starcounter app shell renders the view-model if...